app/_src/guides/meshidentity-guide.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Issuing Identity with MeshIdentity
+---
+
+{% assign kuma = site.mesh_install_archive_name | default: "kuma" %}
+{% assign kuma-system = site.mesh_namespace | default: "kuma-system" %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign kuma-control-plane = kuma | append: "-control-plane" %}
+
+The [MeshIdentity](TODO) allows you to issue identity for selected data planes. This approach is [Spiffe](https://spiffe.io/docs/latest/spiffe-about/overview/) compliant and can
+be used with Spire. In this guide we will take a look on how to issue identities using bundled provider.
+
+## Prerequisites
+
+Before you begin, make sure you have the following tools installed:
+
+* [Helm](https://helm.sh/) – used to install and manage Kubernetes applications
+* [minikube](https://minikube.sigs.k8s.io/docs/) – used to run a local Kubernetes cluster for testing
+
+### Start a Kubernetes cluster
+
+Start a local Kubernetes cluster using minikube. The `-p` flag creates a new profile named `mesh-zone`:
+
+```bash
+minikube start -p mesh-zone
+```
+
+{% tip %}
+If you already have a running Kubernetes cluster, either locally or in the cloud (for example, EKS, GKE, or AKS), you can skip this step.
+{% endtip %}
+
+### Install {{ Kuma }}
+
+Install {{site.mesh_product_name}} control plane with Helm by executing:
+
+```sh
+helm repo add {{site.mesh_helm_repo_name}} {{site.mesh_helm_repo_url}}
+helm repo update
+helm install --create-namespace --namespace {{ kuma-system }} {{ site.mesh_helm_install_name }} {{ site.mesh_helm_repo }}
+```
+
+### Deploy demo application
+
+1.  Deploy the application
+    ```sh
+    kubectl apply -f kuma-demo://k8s/000-with-kuma.yaml
+    kubectl wait -n kuma-demo --for=condition=ready pod --selector=app=demo-app --timeout=90s
+    ```
+
+2.  Port-forward the service to the namespace on port 5000:
+
+    ```sh
+    kubectl port-forward svc/demo-app -n kuma-demo 5050:5050
+    ```
+
+3. Try making requests to demo-app
+   ```sh
+   curl -XPOST localhost:5050/api/counter
+   ```
+   You should see similar results:
+   ```
+   {"counter":1,"zone":""}
+   ```
+
+## Concepts
+
+In {{ Kuma }} we define two concepts around identity that need to be well understood:
+
+* **Identity** - (Who a workload is) - A workload's identity is the name encoded in its certificate, and this identity 
+  is considered valid only if the certificate is signed by a Trust.
```

Comment:
You can mention and reference SPIFFE here: 
```suggestion
* **Identity** - (Who a workload is) - A workload's identity is expressed as a [SPIFFE ID](https://spiffe.io/docs/latest/spiffe-about/overview/) (the name encoded in its certificate’s Subject Alternative Name).
```

Also I'm not sure about the second part:

> identity is considered valid only if the certificate is signed by a Trust

you can have no MeshTrust policies at all, but it doesn't mean your identity is invalid. Can we just remove the second part? I think it's already covered by the `Trust` concept.